### DISCUSSION TITLE
Webpack-dev-server

### DISCUSSION THREAD

**Discussion by jenseo** [ixkaito/bathe#19] [Ideas]
Hey, any plans on introducing webpack-dev-server with hot reloading to the project? I kind of miss it tbh, but not sure it would work on a structure like this? If you can give me some hints on what's needed for Bathe to work properly, I could make a PR for it.

Thanks!

**Comment by jenseo** [comment]
Sorry, I feel really stupid now, I just realized there is reloading active with browsersync, somehow I missed that. I just got started developing with your starter, so I guess I had not wrapped my head totally around the structure. Anyways, thanks for a great developer experience. So far it works out real good for me (especially since I realized I had the hot reloading in place :D )

**Reply by ixkaito** [reply]
@jenseo Never mind. Thank you for your comment :D
